acceptance/analyzer_test.go

```diff
@@ -76,317 +77,496 @@ func TestAnalyzer(t *testing.T) {
+				cmd := exec.Command(
+					"docker", "run", "--rm",
+					"--env", "CNB_PLATFORM_API="+apiString,
+					analyzeImage,
+					ctrPath(analyzerPath),
+					"-group", "group.toml",
+					"-previous-image", "some-image",
+				) // #nosec G204
+				output, err := cmd.CombinedOutput()
+
+				h.AssertNotNil(t, err)
+				expected := "flag provided but not defined: -group"
+				h.AssertStringContains(t, string(output), expected)
+			})
 		})
-	})
-
-	when("group path is provided", func() {
-		it("uses the provided group path", func() {
-			h.DockerSeedRunAndCopy(t,
-				containerName,
-				cacheFixtureDir, ctrPath("/cache"),
-				copyDir, ctrPath("/layers"),
-				analyzeImage,
-				h.WithFlags(
-					"--env", "CNB_PLATFORM_API="+latestPlatformAPI,
-				),
-				h.WithArgs(
+
+		when("called with skip layers", func() {
+			it("errors", func() {
+				h.SkipIf(t, api.MustParse(apiString).Compare(api.MustParse("0.7")) < 0, "Platform API < 0.7")
+				cmd := exec.Command(
+					"docker", "run", "--rm",
+					"--env", "CNB_PLATFORM_API="+apiString,
+					analyzeImage,
 					ctrPath(analyzerPath),
-					"-cache-dir", ctrPath("/cache"),
-					"-group", ctrPath("/layers/other-group.toml"),
-					"some-image",
-				),
-			)
-
-			h.AssertPathExists(t, filepath.Join(copyDir, "layers", "some-other-buildpack-id"))
-			h.AssertPathDoesNotExist(t, filepath.Join(copyDir, "layers", "some-buildpack-id"))
+					"-skip-layers",
+					"-previous-image", "some-image",
+				) // #nosec G204
+				output, err := cmd.CombinedOutput()
+
+				h.AssertNotNil(t, err)
+				expected := "flag provided but not defined: -skip-layers"
+				h.AssertStringContains(t, string(output), expected)
+			})
 		})
-	})
-
-	when("analyzed path is provided", func() {
-		it("writes analyzed.toml at the provided path", func() {
-			h.DockerRunAndCopy(t,
-				containerName,
-				copyDir,
-				ctrPath("/some-dir/some-analyzed.toml"),
-				analyzeImage,
-				h.WithFlags("--env", "CNB_PLATFORM_API="+latestPlatformAPI),
-				h.WithArgs(
+
+		when("called with cache dir", func() {
+			it("errors", func() {
+				h.SkipIf(t, api.MustParse(apiString).Compare(api.MustParse("0.7")) < 0, "Platform API < 0.7")
+				cmd := exec.Command(
+					"docker", "run", "--rm",
+					"--env", "CNB_PLATFORM_API="+apiString,
+					analyzeImage,
 					ctrPath(analyzerPath),
-					"-analyzed", ctrPath("/some-dir/some-analyzed.toml"),
-					"some-image",
-				),
-			)
+					"-cache-dir", "/cache",
+					"-previous-image", "some-image",
+				) // #nosec G204
+				output, err := cmd.CombinedOutput()
+
+				h.AssertNotNil(t, err)
+				expected := "flag provided but not defined: -cache-dir"
+				h.AssertStringContains(t, string(output), expected)
+			})
+		})
 
-			assertAnalyzedMetadata(t, filepath.Join(copyDir, "some-analyzed.toml"))
+		when("cache image tag and cache directory are both blank", func() {
+			it("warns", func() {
+				h.SkipIf(t, api.MustParse(apiString).Compare(api.MustParse("0.7")) >= 0, "Platform API >= 0.7")
```

Comment:
```suggestion
				h.SkipIf(t, api.MustParse(apiString).Compare(api.MustParse("0.7")) >= 0, "Platform API >= 0.7 does not warn because it does not accept a -cache-dir flag")
```

Reply:
(maybe too wordy but it helped me understand it)

Reply:
Does this warning move "elsewhere" for newer platform API? Like in restorer?

Reply:
It probably should! But I don't think it does now.